wrapper/src/main/java/software/amazon/jdbc/plugin/efm/HostMonitorImpl.java

```diff
@@ -335,7 +335,6 @@ ConnectionStatus checkConnectionStatus(final long shortestFailureDetectionInterv
 
         LOGGER.finest(() -> "Opening a monitoring connection to " + this.hostSpec.getUrl());
         startNano = this.getCurrentTimeNano();
-        // TODO: replace with ConnectionService#open
```

Comment:
I believe we decided to not adapt the EFM1 monitors because they are old, have been superseded by EFM2, and do not fit the structure of the refactoring well. However, should we change this to use ConnectionService instead of forceConnect?